src/index.js

```diff
@@ -1,8 +1,11 @@
 import ReactDOM from 'react-dom'
 import {Simulate} from 'react-dom/test-utils'
-import {queries, wait} from 'dom-testing-library'
+import {queries, wait, fireEvent} from 'dom-testing-library'
 
-function render(ui, {container = document.createElement('div')} = {}) {
+function render(
+  ui,
+  {container = document.body.appendChild(document.createElement('div'))} = {},
```

Comment:
We definitely don't want to do this right now. We could expose a `renderIntoDocument` method that does this (and removes the node on `unmount` or something). The problem with this is that you could have a bunch of elements dangling out on the body throughout your tests and if you don't clean it up it could break things in unexpected ways. Also it's a source of memory leaks.

As discussed in https://github.com/kentcdodds/react-testing-library/issues/35 we'll have to recommend to people (in the docs) that if they want to use this method, they must either 1) append the `container` to the body themselves or 2) use the `renderIntoDocument` (which I think we should create as a small abstraction over this function which appends the container to the body). In either case people should be warned that they should create a `beforeEach(() => document.body.innerHTML = '')` in their test. Perhaps we could even create a util so they could do: `beforeEach(clearDOM)`.

Thoughts?

Reply:
Works for me. I'd suggest an `unmountAll` helper:

```js
afterEach(unmountAll)
```

Clearing the DOM will clean up the DOM but won't trigger `componentWillUnmount` on components which might have their own cleanup to do.